vue-components/src/components/Lookup.vue

```diff
@@ -86,6 +92,41 @@ export default Vue.extend( {
 			},
 			default: 'medium',
 		},
+		/**
+		 * Sets the value of the Lookup component's inner `<input>` element. This prop can be used with the `.sync`
+		 * modifier. When bound to a field in the consuming component's data object, it can be used within a watcher or
+		 * computed property to dynamically update the Lookup's `menuItems` prop.
+		 */
+		searchInput: {
+			type: String,
+			default: '',
+		},
+	},
+
+	methods: {
+		onInput( value: string ): void {
```

Comment:
see comment about loading indication behaviour, vuetify does this with a [`loading` prop](https://github.com/vuetifyjs/vuetify/blob/master/packages/vuetify/src/mixins/loadable/index.ts), example: https://vuetifyjs.com/en/components/autocompletes/#cryptocurrency-selector (not sure if that's the way to go though, it's really a design question).